XamarinCommunityToolkitSample/Microsoft.Toolkit.Xamarin.Forms.Sample.csproj

```diff
@@ -42,4 +41,9 @@
   <ItemGroup>
     <ProjectReference Include="..\XamarinCommunityToolkit\Microsoft.Toolkit.Xamarin.Forms.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Update="Pages\Behaviors\MaskedBehaviorPage.xaml.cs">
+      <DependentUpon>MaskedBehaviorPage.xaml</DependentUpon>
+    </Compile>
+  </ItemGroup>
```

Comment:
This file changes look suspect for me. I guess something wrong here..

Reply:
Ye I noticed that before creating the PR. But when I tried to change it back I get this error in the sample project - 
![image](https://user-images.githubusercontent.com/6544051/89161296-52c34500-d56a-11ea-8a49-6e485fb1a85d.png)